### PR TITLE
Adding Ember Inspector link to debugging.md

### PR DESCRIPTION
**Pull request by jbryson3**
This should address #1574. Originally I had written a blurb on the features of ember inspector and included the install links until I realized that we had a whole section dedicated to it. So I slimmed down the description and included a link to the existing guide.

I felt like the Ember inspector was important enough to be included at the top of the debugging section. IMO, its the equivalent of "is your computer turned on?"

### CODE REVIEW

File: guides/release/configuring-ember/debugging.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 Ember provides several configuration options that can help you debug problems
 with your application.
 
+## Ember Inspector
+The [Ember Inspector](https://github.com/emberjs/ember-inspector) is a browser extension that makes understanding and debugging your Ember.js application a snap. To learn more about it check out the [dedicated guide](../../ember-inspector/).
```
Comment (ijlee2) [inline guides/release/configuring-ember/debugging.md:5]: Can we update the first sentence to not use the word "snap"? I think the idiom may be unfamiliar outside of North America.

```suggestion
The [Ember Inspector](https://github.com/emberjs/ember-inspector) is a browser extension that makes understanding and debugging your Ember.js application a snap. To learn more, check out the [dedicated guide](../../ember-inspector/).
```

File: guides/release/configuring-ember/debugging.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 Ember provides several configuration options that can help you debug problems
 with your application.
 
+## Ember Inspector
```
Comment (ijlee2) [inline guides/release/configuring-ember/debugging.md:4]: Can we update lines 1 and 2 to something like:

```
Ember provides a browser extension and several configuration options
to help you debug your application.
```

Initially, I was confused by why the issue asked to mention Ember Inspector in a chapter that discussed configuration options. By updating the introductory paragraph, I think it'll make more sense to mention Ember Inspector.

**Review by ijlee2 — CHANGES_REQUESTED** [review]
Hi, @jbryson3. Thanks for working on the issue and making your 1st contribution to the Ember Guides!

I thought your paragraph was a great start. I left a couple of suggestions that may help the chapter flow more nicely. Will you have a look?

If you have questions about my feedback, please feel free to let me know. Thanks again!

File: guides/release/configuring-ember/debugging.md (hunk 2 above)
Comment (jbryson3) [reply guides/release/configuring-ember/debugging.md:4]: Nice catch, I'll make that change

File: guides/release/configuring-ember/debugging.md (hunk 1 above)
Comment (jbryson3) [reply guides/release/configuring-ember/debugging.md:5]: Absolutely, I took that sentence directly from the description of Ember Inspector on the Chrome web store. But I agree about the language. I will update to make it less idiomatic.

**Review by ijlee2 — APPROVED** [review]
✨

**Pull request merged**